Tests/ObjCTests.mm

```diff
@@ -62,7 +62,10 @@ static void checkIt(id obj, const char* json) {
     checkIt(@123456789,     "123456789");
     checkIt(@123456789012,  "123456789012");
     checkIt(@(INT64_MAX),   "9223372036854775807");
-    checkIt(@(UINT64_MAX),  "18446744073709551615");
+    // checkIt(@(UINT64_MAX),  "18446744073709551615");
```

Comment:
Can we remove this line?

Reply:
okay. I was waving between whether to remove it.